conan.lock

```diff
@@ -1,90 +0,0 @@
-{
```

Comment:
I'm ok with deleting this, as it was causing me difficulties too, but could we re-open the ticket so we can investigate how to do it right? Using a lockfile is an important part of reproducible builds, so there should be a way to get it to work right in CI or as part of normal development workflow yielding updates as part of PRs.

Reply:
I've reopened https://viam.atlassian.net/browse/RSDK-10413.